vscode-trace-extension/src/trace-explorer/trace-utils.ts

```diff
@@ -35,11 +35,11 @@ export const zoomHandler = (hasZoomedIn: boolean): void => {
     TraceViewerPanel.zoomOnCurrent(hasZoomedIn);
 };
 
-export const openDialog = async (): Promise<vscode.Uri | undefined> => {
+export const openDialog = async (selectFiles: boolean = false): Promise<vscode.Uri | undefined> => {
```

Comment:
Type boolean inferred from boolean literal. Can change to:
`selectFiles = false`

Reply:
Fixed!